gpiozero/boards.py

```diff
@@ -1686,3 +1687,52 @@ def off(self):
         Turns the socket off.
         """
         self.value = False
+
+
+class JamHat(CompositeOutputDevice):
+    """
+    Extends :class: `CompositeOutputDevice` for the `ModMyPi JamHat` board.
+
+    There are 6 LEDs, two buttons and a tonal buzzer. The pins are fixed.
+
+    Usage:
+        from gpiozero import JamHat
+
+        hat = JamHat()
+        hat.lights_1.on()
+        hat.buzzer.play(2000)
+        hat.off()
+
+    :param bool pwm:
```

Comment:
Given that the JamHat has more than just LEDs, but this argument applies only to the LEDs, I wonder if something like `pwm_leds` might be a better name? :man_shrugging:

Reply:
It's tricky. Though LEDs are the only variable device. I think `pwm` is fine and the docstring is clear enough.